standardization/src/test/scala/za/co/absa/enceladus/standardization/interpreter/StandardizationCsvSuite.scala

```diff
@@ -63,6 +63,14 @@ class StandardizationCsvSuite extends fixture.FunSuite with SparkTestBase with T
 
   /** Creates a dataframe from an input file name path and command line arguments to Standardization */
   private def getTestDataFrame(tmpFileName: String, args: Array[String]): DataFrame = {
+    val cmd: CmdConfig = CmdConfig.getCmdLineArguments(args)
+    StandardizationJob
+      .getFormatSpecificReader(cmd, dataSet, Some(schema))
+      .schema(schema)
+      .load(tmpFileName)
+  }
+
+  private def getTestDataFrameWOmaxColumsAdjustment(tmpFileName: String, args: Array[String]): DataFrame = {
```

Comment:
This ends up being a bit of duplicated code. How about we get rid of this function and add a param to the original one like so:
```
  private def getTestDataFrame(tmpFileName: String, args: Array[String], hasMaxColumns: Boolean = false): DataFrame = {
    val cmd = CmdConfig.getCmdLineArguments(args)
    val schemaOpt = if (hasMaxColumns) Some(schema) else None
    StandardizationJob
      .getFormatSpecificReader(cmd, dataSet, schemaOpt)
```

Reply:
good spot, thank you